api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -1933,6 +1959,8 @@ message PaginatedResource {
     types.AppServerOrSAMLIdPServiceProviderV1 AppServerOrSAMLIdPServiceProvider = 11 [deprecated = true];
     // SAMLIdPServiceProvider represents a SAML IdP service provider resource.
     types.SAMLIdPServiceProviderV1 SAMLIdPServiceProvider = 12 [(gogoproto.jsontag) = "saml_idp_service_provider,omitempty"];
+    // DynamicWindowsDesktop represents a DynamicWindowsDesktop resource.
+    types.DynamicWindowsDesktopV1 DynamicWindowsDesktop = 15 [(gogoproto.jsontag) = "dynamic_windows_desktop,omitempty"];
```

Comment:
Do we need the dynamically-registered version of desktops here? Or will each `DynamicWindowsDesktopV1` get turned into a `WindowsDesktopV3` that will already show up here?

Reply:
You're right we don't need it here, `WindowsDesktopV3` will be enough, removed